packages/office-addin-sso/src/configure.ts

```diff
@@ -9,11 +9,17 @@ import * as defaults from './defaults';
 import * as fs from 'fs';
 import { addSecretToCredentialStore, writeApplicationData } from './ssoDataSettings';
 import { ManifestInfo, readManifestFile } from 'office-addin-manifest';
+import * as usageDataHelper from './usagedata-helper';
+let usageDataInfo: Object = {};
 require('dotenv').config();
 
 export async function configureSSOApplication(manifestPath: string, port: string) {
+    // Log start time for configuration process
+    const startSsoCongiguration = (new Date()).getTime();
```

Comment:
misspelling. I recommmend using `startTime` or `ssoConfigStartTime`. Phrasing as `startSsoConfig` makes it a verb that implies it is going to start something, rather than a noun which indicates a time.